app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -41,6 +41,10 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
+  def self.fonticon
+    @fonticon ||= "pficon-kubernetes".freeze
```

Comment:
@Fryguy but this is the reason I wanted somebody to see it.. 

On one hand, putting UI stuff into models feels a bit weird, on the other, assuming that our `ems_type` will always match an icon name is a bit brittle, and lacks necessary flexibility.

I can do a big case in `listicon_icon` instead, but I'm pretty sure that's worse, in that it breaks container pluggability.